src/common/styles/index.tsx

```diff
@@ -1,2 +1,5 @@
 export { default as GlobalStyles } from "./GlobalStyles"
 export { default as ThemeProvider } from "./ThemeProvider"
+import * as palette from "./palette"
```

Comment:
Can we do
`export { * as palette } from "./palette`

Reply:
[ES6 export](http://www.ecma-international.org/ecma-262/6.0/#table-42) and [TS module](https://www.typescriptlang.org/docs/handbook/modules.html#allvalidatorsts) does not allow to do it that way. Link is what closest they go to your use case. we need all palettes in one object.